### PR TITLE
List structure fo to do list project

### PR DESCRIPTION
**Pull request by yersel500**
I follow the next requirements:

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (div or ul element). The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
  - description [string].
  - completed [bool].
  - index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
  On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,43 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 
-function component() {
-  const element = document.createElement('div');
+// function component() {
+//   const element = document.createElement('div');
```
Comment (FrederickMih) [inline src/index.js]: - [x] Please do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+//   // Lodash, now imported by this script
+//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
```
Comment (FrederickMih) [inline src/index.js]: - [x] As mentioned above, do not commit old pieces of code as an inline comments.

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
 
-  return element;
-}
+//   return element;
+// }
```
Comment (FrederickMih) [inline src/index.js]: - [x] Please let's get rid of commented code.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
### Hi @yersel500 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @yersel500** , 

### **Congratulations! 🎉 Project Approved ✅**

Geat job making changes required  by the previous reviewer :+1: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- Your README looks descriptive :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**